src/App.css

```diff
@@ -1,25 +1,75 @@
 body {
-  background-color: #121212;
-  color: #cacaca;
+  display: flex;
+  flex-direction: column;
+  justify-content: top;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  transition: background 0.1s linear;
+  color: #fff;
+  background-color: #292c35;
 }
 
-.square {
-  width: 150px;
-  height: 150px;
-  border-color: #121212;
-  border-width: 5px;
-  font-size: 30px;
-  border-radius: 20%;
-  background: white;
+.head {
+  text-align: center;
+  opacity: 70%;
 }
 
 .content {
```

Comment:
I observed that your css is all stored in app.css. The className content is good in functionality however, I would recommend that you break your css content into various css files corresponding to the different components. For example .content is connected to lightSwitch component js. Maybe you could create a css file for that specific component classnames and id for better clarity.